packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -408,31 +408,49 @@ test('convertHeaders', () => {
         },
       ],
     },
+    {
+      source: '/blog/:path*',
+      headers: [
+        {
+          key: 'on-blog',
+          value: 'yup',
+        },
```

Comment:
@ijjk @dav-is Do we want to support path segments for key/value?

For example:
```js
{
  source: '/blog/:post',
  headers: [{
    key: 'x-blog-post',
    value: ':post',
  }]
}
```

Reply:
I think `$1` would still work so might as well

Reply:
Double checked `$1` works so makes sense for segments to work too 👍